content/references/rippled-api/public-rippled-methods/account-methods/account_lines.md

```diff
@@ -422,12 +422,12 @@ Each trust line object has some combination of the following fields:
 | `limit_peer`     | String           | The maximum amount of currency that the counterparty account is willing to owe the perspective account |
 | `quality_in`     | Unsigned Integer | Rate at which the account values incoming balances on this trust line, as a ratio of this value per 1 billion units. (For example, a value of 500 million represents a 0.5:1 ratio.) As a special case, 0 is treated as a 1:1 ratio. |
 | `quality_out`    | Unsigned Integer | Rate at which the account values outgoing balances on this trust line, as a ratio of this value per 1 billion units. (For example, a value of 500 million represents a 0.5:1 ratio.) As a special case, 0 is treated as a 1:1 ratio. |
-| `no_ripple`      | Boolean          | _(May be omitted)_ `true` if this account has enabled the [No Ripple flag](rippling.html) for this line. If omitted, that is the same as `false`. |
-| `no_ripple_peer` | Boolean          | _(May be omitted)_ `true` if the peer account has enabled the [No Ripple flag](rippling.html). If omitted, that is the same as `false`. |
-| `authorized`     | Boolean          | _(May be omitted)_ `true` if this account has [authorized this trust line](authorized-trust-lines.html). If omitted, that is the same as `false`. |
-| `peer_authorized`| Boolean          | _(May be omitted)_ `true` if the peer account has [authorized this trust line](authorized-trust-lines.html). If omitted, that is the same as `false`. |
-| `freeze`         | Boolean          | _(May be omitted)_ `true` if this account has [frozen](freezes.html) this trust line. If omitted, that is the same as `false`. |
-| `freeze_peer`    | Boolean          | _(May be omitted)_ `true` if the peer account has [frozen](freezes.html) this trust line. If omitted, that is the same as `false`. |
+| `no_ripple`      | Boolean          | _(May be omitted)_ If `true`, this account has enabled the [No Ripple flag](rippling.html) for this trust line. If present and `false`, this account has disabled the No Ripple flag, but that is not considered [the default state](ripplestate.html#contributing-to-the-owner-reserve) because the account has the Default Ripple flag enabled. If omitted, the account has the No Ripple flag disabled for this trust line and Default Ripple disabled. [Updated in: rippled 1.7.0][] |
+| `no_ripple_peer` | Boolean          | _(May be omitted)_ If `true`, the peer account has enabled the [No Ripple flag](rippling.html) for this trust line. If present and `false`, this account has disabled the No Ripple flag, but that is not considered [the default state](ripplestate.html#contributing-to-the-owner-reserve) because the account has the Default Ripple flag enabled. If omitted, the account has the No Ripple flag disabled for this trust line and Default Ripple disabled. [Updated in: rippled 1.7.0][] |
```

Comment:
Same suggestion for the second sentence here:

> If present and `false`, this account has disabled the No Ripple flag, but, because the account also has the Default Ripple flag enabled, that is not considered [the default state](ripplestate.html#contributing-to-the-owner-reserve).